nodeJS/express/installing_postgresql.md

```diff
@@ -0,0 +1,293 @@
+<!-- markdownlint-disable MD024 TOP004 -->
+
+### Introduction
+
+Data persistence is integral to the vast majority of web applications. Data persistence is achieved through databases. Being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.
+
+For example, consider The Odin Project. Where is your username stored? Where are project submissions stored? Where does this lesson content, the one you're reading right now, get stored? Yes, a database!
+
+We've chosen [PostgreSQL](https://www.postgresql.org/) as our database of choice for the curriculum. All SQL knowledge that you've learned in the earlier course should be transferable to PostgreSQL. We'll also learn new PostgreSQL specific things. Watch through this [fun and short introduction video to PostgreSQL by Fireship](https://www.youtube.com/watch?v=n2Fluyr3lbc).
+
+<div class="lesson-note lesson-note--critical" markdown="1">
+
+#### Prerequisite SQL knowledge
+
+Make sure you've completed the [SQL course](https://www.theodinproject.com/paths/full-stack-javascript/courses/databases).
+
+This lesson and all subsequent lessons will assume you understand SQL syntax and concepts.
+
+</div>
+
+In this lesson, we'll install PostgreSQL and in the next lesson we'll learn how to use it in our Express application.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- How to install PostgreSQL.
+
+### Installing PostgreSQL
+
+Pick your operating system below to get the appropriate steps for installing PostgreSQL.
+
+<details markdown="block">
+
+<summary class="dropDown-header">Linux</summary>
+
+### Step 1: Make sure the system is up to date
```

Comment:
The headings in these steps would need to be increased by one level since these are subsections of the larger "Installing PostgreSQL" section. Right now it would come off as this Step 1 section being more separate from "Installing" when it really isn't.

Reply:
(see above comment)